src/list/watch_history.rs

```diff
@@ -13,14 +17,14 @@ pub struct WatchHistoryEntry {
     #[default(DateTime::<Utc>::from(std::time::SystemTime::UNIX_EPOCH))]
     pub date_played: DateTime<Utc>,
     pub playhead: u32,
-    pub fullywatched: bool,
+    pub fully_watched: bool,
 
     /// Should always be [`MediaCollection::Episode`] or [`MediaCollection::Movie`].
     pub panel: MediaCollection,
 }
 
 #[allow(dead_code)]
-#[derive(Clone, Debug, Deserialize, smart_default::SmartDefault, Request)]
+#[derive(Clone, Debug, Deserialize, smart_default::SmartDefault)]
```

Comment:
The removal of `Request` and the manual implementation below is not necessary here. `#[request(executor(panel))]` can be passed as separate macro attribute that handles the implementation process.

```suggestion
#[derive(Clone, Debug, Deserialize, smart_default::SmartDefault, Request)]
#[request(executor(panel))]
```

Reply:
Ah got it! I'm not super familiar with macros so this was my best attempt! I'll give the suggestion a try locally and update the PR when the change is implemented! Appreciate all the amazing work being done with this library!

Reply:
Thanks and no prob, there isn't documentation about this attribute either :)